pulsar-grpc/src/main/java/org/apache/pulsar/grpc/Constant.java

```diff
@@ -0,0 +1,9 @@
+package org.apache.pulsar.grpc;
```

Comment:
the license headers are missing. you can run 'mvn license:format' to attach license headers.

Reply:
:+1 adding to todo

Reply:
done